Client/Frontend/Onboarding/Models/IntroViewModel.swift

```diff
@@ -44,4 +58,15 @@ class IntroViewModel: OnboardingViewModelProtocol, FeatureFlaggable {
     func saveHasSeenOnboarding() {
         introScreenManager?.didSeeIntroScreen()
     }
+
+    // MARK: SkAdNetwork
+    // this event should be sent in the first 24h time window, if it's not sent the conversion value is locked by Apple
+    func sendOnboardingUserActivationEvent() {
```

Comment:
nit: I might call this "update" rather than "send", since we don't control the sending
```suggestion
    func updateOnboardingUserActivationEvent() {
```